.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   GO_VERSION: '1.18'
-  ACTION_MSRV_TOOLCHAIN: 1.59.0
+  ACTION_MSRV_TOOLCHAIN: 1.60.0
```

Comment:
is this toolchain available on rhcos?

Reply:
cc @cgwalters if you know

Reply:
We hit this problem with the PR: https://github.com/rust-lang/cargo/issues/10623

The only thing I could do is bump the version.